scripts/judge_extended_stack.py

```
@@ -326,6 +447,71 @@ def run_job(
     path.write_text(json.dumps(record, ensure_ascii=False, indent=2))
 
 
+def _running_jobs() -> set[tuple[str, str]]:
+    """이 기계에서 지금 돌고 있는 (구성, 전략) 작업. 드라이버가 죽어도 남은 작업자를 센다."""
+    listing = subprocess.run(
+        ["ps", "-axo", "command"], capture_output=True, text=True, check=False
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)